etc/releasing.md

```diff
@@ -671,6 +671,20 @@ Add a section for the next patch release, e.g. following a `1.2.3` release:
 
 Commit the changes to the `releases/vX.Y` branch and push the branch to the remote repository (a PR is not required for this step).
 
+Checkout the `post-release-changes` branch.
+
+Add a section for the new patch release containing the same entries as in the release, e.g. following a `1.2.3` release:
+
+```md
+## 1.3.0 [Unreleased]
```

Comment:
```suggestion
## 1.2.4 [Unreleased]
```

Reply:
1.3.0 is correct because this code block corresponds to the `CHANGELOG.md` on `master`, not the release branch (as in earlier instructions above).

Do we expect / want to maintain patch release notes on `master` prior to release? IIRC we avoid doing so to avoid potential cherry-pick conflicts with the release branch.

Reply:
On further thought, I believe the expectation is yes (with caveats) given the Important note added in https://github.com/mongodb/mongo-cxx-driver/pull/1161 concerning shifting of entries from an unreleased patch section -> next unreleased minor section.

Revised the instructions accordingly.